store/state.go

```diff
@@ -15,6 +16,31 @@ import (
 	"github.com/rqlite/rqlite/v8/snapshot"
 )
 
+// IsStaleRead returns whether a read is stale.
+func IsStaleRead(
+	LeaderLastContact time.Time,
```

Comment:
It is strange to see PascalCase for local variables.

Reply:
Yeah, Copilot did it. :-) Will fix now, you're right.